### DISCUSSION TITLE
Workflow actions not running on personal account

### DISCUSSION THREAD

**Discussion by erenkurnaz** [community/community#26691] [Actions]
<p>Greetings, I have some issue with workflows and I cannot fix that.<br>
<strong>When I run the workflow mentioned below in my account, it has no effect. I tried same actions on my friends account it works as expected</strong></p>
<pre><code class="lang-auto">name: GitHub Actions Demo
on:
  workflow_dispatch:
  push:
    branches: [ main ]
  pull_request:
    branches: [ main ]
jobs:
  Explore-GitHub-Actions:
    runs-on: ubuntu-latest
    steps:
      - run: echo "🎉 The job was automatically triggered by a ${{ github.event_name }} event."
      - run: echo "🐧 This job is now running on a ${{ runner.os }} server hosted by GitHub!"
      - run: echo "🔎 The name of your branch is ${{ github.ref }} and your repository is ${{ github.repository }}."
      - name: Check out repository code
        uses: actions/checkout@v3
      - run: echo "💡 The ${{ github.repository }} repository has been cloned to the runner."
      - run: echo "🖥️ The workflow is now ready to test your code on the runner."
      - name: List files in the repository
        run: |
          ls ${{ github.workspace }}
      - run: echo "🍏 This job's status is ${{ job.status }}."
</code></pre>
<h1>
<a name="my-result-1" class="anchor" href="#my-result-1"></a>My result:</h1>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/f/b/fba3d8c4130e43549ffe465f203d265cffd73ee5.png" data-download-href="/uploads/short-url/zU6SSm3DEmuMFoEhI4TbLXxlc1L.png?dl=1" title="Screen Shot 2022-04-25 at 15.17.07.png" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/24798543/181099243-2d199d95-4bde-486a-89e7-36db1bbbaa28.png" alt="Screen Shot 2022-04-25 at 15.17.07.png" data-base62-sha1="zU6SSm3DEmuMFoEhI4TbLXxlc1L" width="690" height="383" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/f/b/fba3d8c4130e43549ffe465f203d265cffd73ee5_2_690x383.png, https://aws1.discourse-cdn.com/github/optimized/3X/f/b/fba3d8c4130e43549ffe465f203d265cffd73ee5_2_1035x574.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/f/b/fba3d8c4130e43549ffe465f203d265cffd73ee5.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/f/b/fba3d8c4130e43549ffe465f203d265cffd73ee5_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">Screen Shot 2022-04-25 at 15.17.07.png</span><span class="informations">1139×633 64.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<h1>
<a name="my-friends-result-2" class="anchor" href="#my-friends-result-2"></a>My friends result</h1>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/9/a/9a3068d33d679405bedb9b293c5bb9f1bf91ff54.png" data-download-href="/uploads/short-url/m01fkuIvqoLGEJ0LxEDsg4XGdKs.png?dl=1" title="Screen Shot 2022-04-25 at 15.22.47.png" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/24798543/181099245-2f6f759d-fad7-487d-8f6b-8487d1dc7777.png" alt="Screen Shot 2022-04-25 at 15.22.47.png" data-base62-sha1="m01fkuIvqoLGEJ0LxEDsg4XGdKs" width="690" height="388" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/9/a/9a3068d33d679405bedb9b293c5bb9f1bf91ff54_2_690x388.png, https://aws1.discourse-cdn.com/github/optimized/3X/9/a/9a3068d33d679405bedb9b293c5bb9f1bf91ff54_2_1035x582.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/9/a/9a3068d33d679405bedb9b293c5bb9f1bf91ff54.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/9/a/9a3068d33d679405bedb9b293c5bb9f1bf91ff54_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">Screen Shot 2022-04-25 at 15.22.47.png</span><span class="informations">1246×702 84.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by erenkurnaz** [comment]
<p>I changed my username from herenkurnaz to erenkurnaz in the past weeks. I forgot to add this detail.</p>

**Comment by Mara-Li** [comment]
<p>Hello ! Someone trying to fork one of my repo have the same issue. Did you found a solution ?</p>

**Comment by airtower-luna** [comment]
<p>Does the branch where you pushed the workflow match the branch filter?</p>

**Comment by erenkurnaz** [comment, marked as answer]
<p>Hello, when the methods I tried did not work, I contacted Github Support. They solved the problem with the development team. If you are having a similar problem, I suggest you contact the Support team.</p>
